airbyte-integrations/connectors/source-slack/source_slack/manifest.yaml

```diff
@@ -253,11 +253,75 @@ definitions:
               - channel_id
             value: "{{ stream_partition.get('channel') }}"
 
+  threads_stream:
+    $ref: "#/definitions/stream_base"
+    $parameters:
+      name: threads
+      path: conversations.replies
+      extractor_field_path: messages
+    primary_key:
+      - channel_id
+      - ts
+    retriever:
+      $ref: "#/definitions/retriever"
+      requester:
+        $ref: "#/definitions/requester"
+        request_parameters:
+          channel: "{{ stream_slice['parent_slice']['channel'] }}"
+        error_handler:
+          type: DefaultErrorHandler
```

Comment:
I saw in the code in `streams.py:SlackStream.get_error_handler()` that the max_retries is set to `20`, should we do the same here?

Reply:
added